wp/plugin/wp.ml

```diff
@@ -18,149 +18,158 @@ module Typ = Extension.Type
-    ~doc:"Amount of times to unroll each loop. By default, WP will unroll \
-          each loop 5 times."
+    ~doc:{|Replaces the default number of times to unroll each loop. WP will
+           unroll each loop 5 times by default.|}
 
 let gdb_output = Cmd.parameter Typ.(some string) "gdb-output"
-    ~doc:"In the case WP determines input registers that result in a refuted \
-          goal, this flag outputs a gdb script to the filename specified. \
-          This script file sets a breakpoint at the the start of the \
-          function being analyzed, and sets the registers and memory to the \
-          values specified in the countermodel, allowing GDB to follow the \
-          same execution trace."
+    ~doc:{|When WP results in SAT, outputs a gdb script to the filename
+           specified. From within gdb, run `source filename.gdb' to set a
+           breakpoint at the function given by `--func' and fill the
+           appropriate registers with the values found in the countermodel. In
+           the case WP returns UNSAT or UNKNOWN, no script will be outputted.|}
 
 let bildb_output = Cmd.parameter Typ.(some string) "bildb-output"
-    ~doc:"In the case WP determines input registers that result in a refuted \
-          goal, this flag outputs a BilDB YAML file to the filename \
-          specified. This file sets the registers and memory to the values \
-          specified in the countermodel, allowing BilDB to follow the same \
-          execution trace."
+    ~doc:{|When WP results in SAT, outputs a BILDB initialization script to the
+           filename specified. This YAML file sets the registers and memory to
+           the values found in the countermodel, allowing BILDB to follow the
+           same execution trace. In the case WP returns UNSAT or UNKNOWN, no
+           script will be outputted.|}
 
 let use_fun_input_regs = Cmd.flag "use-fun-input-regs"
-    ~doc:"If set, at a function call site, uses all possible input registers \
-          as arguments to a function symbol generated for an output register \
-          that represents the result of the function call. If this flag is not \
-          set, no registers will be used."
+    ~doc:{|At a function call site, uses all possible input registers as
+           arguments to a function symbol generated for an output register. If
+           this flag is not present, no registers will be used.|}
 
 let mem_offset = Cmd.flag "mem-offset"
-    ~doc:"If set, maps the symbols in the data and bss sections from their \
-          addresses in the original binary to their addresses in the \
-          modified binary."
+    ~doc:{|This flag is only used in a comparative analysis. Maps the symbols
+           in the data and bss sections from their addresses in the original
+           binary to their addresses in the modified binary. If this flag is
+           not present, WP assumes that memory between both binaries are
+           equal.|}
```

Comment:
Instead of saying "assumes that memory between both binaries are equal", maybe say "assumes that memory between both binaries start at the same offsets"?